_getting-started/intro.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: default
+title: Intro to OpenSearch
+nav_order: 2
+has_math: true
+redirect_from: 
+ - /intro/
+---
+
+# Introduction to OpenSearch
+
+OpenSearch is a distributed search and analytics engine that supports various use cases, from implementing a search box on a website to analyzing security data for threat detection. The term _distributed_ means you can run OpenSearch on multiple computers. _Search and analytics_ means you can search and analyze your data once you ingest it into OpenSearch. Whether your data is geographic or genetic, you can store and analyze it using OpenSearch.
+
+## Document
+
+A _document_ is a unit that stores information (text or structured data). In OpenSearch, documents are stored in [JSON](https://www.json.org/) format. 
+
+You can think of a document in several ways:
+
+- In a database of students, a document might represent one student.
+- When you search for information, OpenSearch returns documents related to your search.
+- If you're familiar with traditional databases, a document represents a row.
+
+For example, in a school database, a document might represent one student and contain the following data.
+
+ID | Name | GPA | Graduation year | 
+:--- | :--- | :--- | :--- | 
+1 | John Doe | 3.89 | 2022 | 
+
+Here is what this document looks like in JSON format:
+
+```json
+{
+  "name": "John Doe",
+  "gpa": 3.89,
+  "grad_year": 2022
+}
+```
+
+You'll learn about how document IDs are assigned in [Indexing documents]({{site.url}}{{site.baseurl}}/getting-started/communicate/#indexing-documents).
+
+## Index
+
+An _index_ is a collection of documents. 
+
+You can think of an index in several ways:
+
+- If you have a collection of encyclopedia articles, an index represents the whole collection.
+- When you search for information, you query data contained in an index.
+- If you're familiar with traditional databases, a document represents a database table.
+
+For example, in a school database, an index might contain all students in the school.
+
+ID | Name | GPA | Graduation year 
+:--- | :--- | :--- | :--- 
+1 | John Doe | 3.89 | 2022
+2 | Jonathan Powers | 3.85 | 2025
+3 | Jane Doe | 3.52 | 2024
+
+## Clusters and nodes
+
+OpenSearch is designed to be a distributed search engine. OpenSearch can run on one or more _nodes_---servers that store your data and process search requests. An OpenSearch *cluster* is a collection of nodes. 
```

Comment:
```suggestion
OpenSearch is designed to be a distributed search engine, meaning that it can run on one or more _nodes_---servers that store your data and process search requests. An OpenSearch *cluster* is a collection of nodes. 
```